extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -94,6 +94,14 @@ tags: {  # <amp-story>
     name: "title"
     mandatory: true
   }
+  attrs: {
+    name: "supports-landscape"
+    value: ""
+  }
+  attrs: {
+    name: "supports-portrait"
```

Comment:
I think we should hold off on adding this to the validation. I think it's better to add it when (if) we add validation tests & logic on the same PR.

Reply:
+1 I think it will give us more flexibility down the line to add this to validation only when we need it

Reply:
Validator team prefers to make things more permissive over time and not more restrictive. So I agree with holding off on adding something that is unsupported if it only gets removed later.